tests/conftest.py

```diff
@@ -183,8 +188,35 @@ def openreview_context(scope="function"):
         }
 
 
-if __name__ == "__main__":
+@pytest.fixture(scope="session")
+def celery_config():
+    return {
+        # 'broker_url': 'amqp://openreview:openreview@localhost:5672/localhost',
```

Comment:
can you remove this line that is not being used?

Reply:
Done.